src/main/resources/avro/bdg.avdl

```diff
@@ -883,57 +884,141 @@ record TranscriptEffect {
 }
 
 /**
- Annotations of a variant, as produced by a tool such as SnpEff or Ensembl VEP.
+ Variant annotation.
  */
 record VariantAnnotation {
 
   /**
    Variant for this annotation.
    */
-  union { null, Variant } variant;
+  union { null, Variant } variant = null;
 
   /**
-   Zero or more transcript effects, per alternate allele and transcript (or other feature).
+   Ancestral allele, VCF INFO reserved key AA, Number=1, shared across all alleles
+   in the same VCF record.
    */
-  array<TranscriptEffect> transcriptEffects = [];
-}
+  union { null, string } ancestralAllele = null;
 
-/**
- Database variant annotations.
- */
-record DatabaseVariantAnnotation {
-  union { null, Variant } variant;
+  /**
+   Allele count, VCF INFO reserved key AC, Number=A, split for multi-allelic sites.
+   */
+  union { null, int } alleleCount = null;
+
+  /**
+   Total read depth, VCF INFO reserved key AD, Number=R, split for multi-allelic
+   sites.
+   */
+  union { null, int } readDepth = null;
+
+  /**
+   Forward strand read depth, VCF INFO reserved key ADF, Number=R, split for
+   multi-allelic sites.
+   */
+  union { null, int } forwardReadDepth = null;
+
+  /**
+   Reverse strand read depth, VCF INFO reserved key ADR, Number=R, split for
+   multi-allelic sites.
+   */
+  union { null, int } reverseReadDepth = null;
+
+  /**
+   Minor allele frequency, VCF INFO reserved key AF, Number=A, split for multi-allelic
+   sites. Use this when frequencies are estimated from primary data, not calculated
+   from called genotypes.
+   */
+  union { null, float } alleleFrequency = null;
+
+  /**
+   Total number of alleles in called genotypes, VCF INFO reserved key AN, Number=1.
+   */
+  //union { null, int } allelesInCalledGenotypes = null;
+
+  /**
+   RMS base quality, VCF INFO reserved key BQ, Number=1.
+   */
+  //union { null, float } rmsBaseQuality = null;
+
+  /**
+   CIGAR string describing how to align an alternate allele to the reference
+   allele, VCF INFO reserved key CIGAR, Number=A, split for multi-allelic sites.
+   */
+  union { null, string } cigar = null;
+
+  /**
+   Membership in dbSNP, VCF INFO reserved key DB, Number=0. Until Number=A and
+   Number=R flags are supported by the VCF specification, this value is shared
+   across all alleles in the same VCF record.
+   */
+  union { null, boolean } dbSnp = null;
+
+  /**
+   Combined depth across samples, VCF INFO reserved key DP, Number=1.
+   */
+  //union { null, int } combinedDepth = null;
```

Comment:
If merging across VCF files, as in @jpdna's use case, this value needs to be updated as new samples are added.  That means VariantAnnotation records aren't immutable.